Add readable terms for the operators

beerose opened the pull request:
### Description
In Browse rows, there were listed postgres operators. This PR adds readable english terms and the operator in brackets so that anybody can understand it quickly.

<img width="178" alt="Screenshot 2020-01-14 at 11 38 03" src="https://user-images.githubusercontent.com/9019397/72337150-596b9700-36c2-11ea-868d-c3a5938d344a.png">


### Affected components

- [x] Console

#### Breaking changes

- [x] No Breaking changes

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/hasura/graphql-engine?pullRequest=3699) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/hasura/graphql-engine?pullRequest=3699) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/hasura/graphql-engine?pullRequest=3699) it.</sub>

coco98 commented:
@beerose @rikinsk Should we change `ilike` to `like (ignore case)` or `like (case insensitive)`?

beerose commented:
Yeah, it makes sense since we want to make it much more readable. I think I'd rather go with `(case insensitive)` since it's more common. 

But take into account that it would quite widen the dropdown:

<img width="335" alt="Screenshot 2020-01-14 at 12 19 43" src="https://user-images.githubusercontent.com/9019397/72340269-2debab00-36c8-11ea-9ce7-1a37b3c6e8da.png">

beerose commented:
I've changed dropdown options to be of the format `[<graphql equivalent operator>] logical name` and added `(case insensitive)` to `ilike` and `nilike`.

I also added the default value `%%` when `like`, `nlike`, `ilike` or `nilike` is chosen. Small demo:
![likes](https://user-images.githubusercontent.com/9019397/72471789-9c715b80-37e3-11ea-8aaf-8322342007b5.gif)

rikinsk commented on console/src/components/Services/Data/constants.js:
```diff
@@ -1,18 +1,22 @@
+  { name: 'not like', value: '$nlike', graphqlOp: '_nlike' },
+  { name: 'like (case-insensitive)', value: '$ilike', graphqlOp: '_ilike' },
+  {
+    name: 'not ilike (case-insensitive)',
```
`not ilike` -> `not like`

rikinsk commented on console/src/components/Services/Data/constants.js:
```diff
@@ -1,18 +1,22 @@
+    graphqlOp: '_nilike',
+  },
+  { name: 'similar', value: '$similar', graphqlOp: '_similar' },
+  { name: 'not similar', value: '$nsimilar', graphqlOp: '_similar' },
```
`_similar` -> `_nsimilar`

beerose commented:
I handled the `$in` and `$nin` case. Also, in the future, we could think about adding validation -- validate if, for expected integer, there's a number, for expected array there's a valid array, etc.
